examples/datacalc.py

```diff
@@ -162,4 +163,4 @@ def calculateSum(self):
             result += value
         else:
             self.viewerFrame.numberLabel.setText(f"sum: {result}")
-            self.broadcast("log", f"Sum: {result}.")
+            logger.log("Sum: %f.", result)
```

Comment:
Why did you use `log()` instead of `info()` here?

Reply:
My mistake..